### ISSUE TITLE
Mesh bed leveling way off 

### ISSUE THREAD

**Issue by airexes** [MarlinFirmware/Marlin#11162]
Using marlin 1.1.8 and a cr10s. I have two identically setup printers and one is exhibiting a strange issue with the manual mesh bed leveling on my bed with a mirror tile attached. I had the printer working fantastic before I changed the hotend mount and made a few firmware changes to compensate. I have made sure the the hotend is level with the crossbar.

I can repeatedly level the corners without issue, however when I start to perform the manual mesh bed leveling everything goes downhill. The first position already needs about -.05 adjustment. The right corner needs about +.2, the middle about -.2 meaning that is a 0.4mm difference . The most deviation I normally saw was 0.075.

So basically when leveling, the left of the bed now has the nozzle placed too high and the right way too low. 

When I actually go to print the left of the bed prints a lot lower then the right. The right side the filament barely sticks. I can feel the z axis compensating like always. 

I then go to check that the corners are still level and they check out perfectly.

**Comment by forkoz** [comment]
Print your mesh. Maybe you have some mechanical issue now. Nothing about 1.1.8 has changed in a while.

**Comment by thinkyhead** [comment]
Please test with the latest `bugfix-1.1.x` (and/or `bugfix-2.0.x`) branch to see where it stands. If the problem has been resolved then we can close this issue. If the issue isn't resolved yet, then we should investigate further.

**Comment by airexes** [comment]
I have tested with the 1.1.x bugfix branch and there was no difference. I went through and only chose a few basic settings I needed to perform the test. I really didn't feel like it was a software issue but didn't know where else to turn.

I've just verified it can't be the z axis by both replacing the z switch and parking the nozzle at a corner and verifying it's level. I raised the z axis 200mm and back down again and it was still perfectly level on that corner.

**Comment by airexes** [comment]
Okay, I replaced the motherboard with a stock Cr10S one. On the stock firmware for some reason I couldn't find mesh bed leveling, but it did have corner leveling utility. I ran it, leveled the corners then homed the machine.
Everything was as it should be for the first time, it actually remained level. I unplugged the printer to see if it would stay level and sure enough it did.

Opened marlin 1.1.8, flashed it. Back to the same problem in exactly the same way. When running the corner leveling utility Left is too high right is too low by the exact same amount. I ran the mesh bed leveling utility and every spot on the bed requires the same compensation as before. 

I can level the corners again with the thumbscrews. Home and again, the nozzle is .1mm to high on the left and .175 too low on the right.

Also I had mentioned I had two. The other one was working okay but now they both have exactly the same issue. I tried swapping control boxes between them and got the same issue.

EDIT: I removed the second z axis rod and unplugged the motor. I can level all corners fine. When I home and go to check if they are level again the nozzle is now 5mm above the bed!!!

**Comment by airexes** [comment]
Solved the issue. I had tightened the 6 wheels for the y axis and everything seemed secure. It took a lot of force to move the bed so I thought everything was okay. This is how it always was so I thought it was normal. Turns out the bearing were slipping in the wheels just a tiny bit. It doesn't make sense to me why they have the wheels layed out horizontally, a dual vertical y axis design would put much less load on the wheels.

I was really hesitant to spend more on this thing and upgrade wheels were expensive but I got them in today. Everything is fine now. I only have to adjust .025mm at the most now. This should also solve my parts becoming very weak in certain spots as the layer adhesion was poor.

The reason it seemed like it was firmware was because the stock cr10s homes at a very slow speed which was enough for the bed to stay level when being knocked against the y axis sensor, the 1.1.7 pre setup hex file had the homing at stock marlin speeds which was enough sometimes, and my 1.1.8 file had the speeds higher which resulted in the bed shifting the same way repeatedly. 

Somehow both my printers developed the issue at nearly the same time.
